packages/orbit-components/src/Breadcrumbs/BreadcrumbsItem/index.tsx

```diff
@@ -1,63 +1,48 @@
 "use client";
 
 import * as React from "react";
-import styled, { css } from "styled-components";
+import cx from "clsx";
 
-import defaultTheme from "../../defaultTheme";
-import ChevronForward from "../../icons/ChevronForward";
 import type { Props } from "./types";
+import ChevronForward from "../../icons/ChevronForward";
 
-const StyledBreadcrumbsItem = styled.li`
-  display: flex;
-  align-items: center;
-`;
-
-const StyledBreadcrumbsItemAnchor = styled(({ component, children, isClickable, ...props }) => {
-  const Component = isClickable ? component : "div";
-  return <Component {...props}>{children}</Component>;
-})`
-  ${({ theme, $active, isClickable }) => css`
-    font-weight: ${$active ? theme.orbit.fontWeightBold : theme.orbit.fontWeightMedium};
-    color: ${theme.orbit.paletteInkDark};
-    text-decoration: ${isClickable && !$active ? "underline" : "none"};
-
-    ${isClickable &&
-    css`
-      transition: color ${theme.orbit.durationFast} ease-in-out;
-      cursor: pointer;
-      border: none;
-      padding: 0;
-      outline-offset: 1px;
-
-      &:hover {
-        text-decoration: none;
-        color: ${theme.orbit.paletteProductNormalHover};
-      }
-    `};
-  `}
-`;
+const Component = ({
+  href,
+  onClick,
+  component: Comp,
+  children,
+  ...props
+}: Props &
+  Pick<
+    React.HTMLAttributes<HTMLDivElement | HTMLAnchorElement | HTMLButtonElement>,
+    "itemScope" | "itemType" | "itemProp" | "itemID" | "className"
+  >) => {
+  const isClickable = onClick != null || href != null;
+  if (!isClickable) return <div {...props}>{children}</div>;
 
-StyledBreadcrumbsItemAnchor.defaultProps = {
-  theme: defaultTheme,
-};
+  if (Comp != null)
+    return (
+      <Comp {...props} onClick={onClick} href={href}>
+        {children}
+      </Comp>
+    );
 
-const StyledBreadcrumbsItemIcon = styled(ChevronForward)`
-  ${({ theme }) => css`
-    margin: 0 ${theme.orbit.spaceXXSmall};
-  `}
-`;
+  if (href == null)
+    return (
+      <button type="button" {...props} onClick={onClick}>
+        {children}
+      </button>
+    );
 
-StyledBreadcrumbsItemIcon.defaultProps = {
-  theme: defaultTheme,
+  return (
+    <a href={href} {...props}>
```

Comment:
I think we should pass onClick here as well. An anchor can have both, right? 🤔

Reply:
yes, you are right 👍🏻